randovania/data/json_data/prime3.txt

```diff
@@ -1470,9 +1696,23 @@ Asset id: 17945996104959776291
       Trivial
 
 > Room Center; Heals? False
+  > Dock to North Jungle Court
+      After Machineworks Bridge Jungleside
+  > Dock to Colossus Vista
+      After Machineworks Bridge Cliffside
   > Pickup (Energy Tank)
       Use Screw Attack (Space Jump)
 
+> Event - Bridge (Cliffside); Heals? False
+  * Event Machineworks Bridge Cliffside
+  > Dock to Colossus Vista
+      Trivial
+
+> Event - Bridge (Jungleside); Heals? False
+  * Event Machineworks Bridge Jungleside
+  > Dock to North Jungle Court
+      Trivial
+
 ----------------
 North Jungle Court
 Asset id: 5310258251750100868
```

Comment:
Dock to Generator Hall North connecting to North Jungle Hall should have After North Jungle Court Turret